### ISSUE TITLE
Toolbar images not loading v14.0.0.0 of ReportViewer

### ISSUE THREAD

**Issue by felixaballi** [Microsoft/Reporting-Services#22]
There are several issues in Stackoverflow depicted:

https://stackoverflow.com/questions/34849286/rdcl-report-viewer-toolbar-icons-not-visible-asp-net
https://stackoverflow.com/questions/39891918/ssrs-2016-report-viewer-v13-toolbar-icons-not-displaying

Perhaps you could look around, it doesn't seems to be a great deal and people'd appreciate it.
I fall down seamlessly and installed it a few days ago (June/2017).

Thanks in advanced.

**Comment by figuerres** [comment]
as of now i am not seeing this problem, @felixaballi   are you seeing this issue right now ?

depending on what you are doing there might be a reason but you would need to give more details on your case.

**Comment by felixaballi** [comment]
Yes, the problem exists (ReportViewer 14.0.0.0). I downloaded it a few days ago.
I fixed the problem with the second article workaround, but it should be contained in the library.

Environment: 
Windows 10 Pro, VS 2017 Professional

**Comment by figuerres** [comment]
this only happens with a report view in a child page that has a master page ?
i had issues with a report view called in an iframe   and had to mess with how i loaded the iframe
but i am not doing a master page  in my app.

i will see if i can report and post back but it may be a few days.....  may be something about the ajax calls that load the inline stuff ....

**Comment by jtarquino** [comment]
There is a new update to the control that should fix the issues you have experienced https://blogs.msdn.microsoft.com/sqlrsteamblog/2017/07/27/updated-report-viewer-control-now-generally-available/

**Comment by Hasanajouz** [comment]
I think this issue is not fixed yet. I tried the latest November update and the icons will not show:
Microsoft.ReportingServices.ReportViewerControl.WebForms 140.1000.523

**Comment by SandarKhin** [comment]
I also used ReportViewer14 with .Net 4.5.2 and happening some problem on reportviewer toolbar.
My case is appearing A,B,C,D instead of print, export, next , previous icons.
 And I got font-face error on ie-11 console for ReportingServicesGlyphs.ttf about permission must be installable.Currently, I am confusing which one makes A,B,C,D icon on report toolbars.
I am working on Window10 , ReportViewer14 , .Net 4.5.2 , VS-2017 and IE11.
I really need the help to solve it. Anyone help? Thank you.
